commit d670a91749c4932649334aeca4d006b9464499e1 (HEAD -> refs/heads/master)
Author: Andrew Wang <andrew.wang@cloudera.com>
Date:   Thu Oct 8 17:07:29 2015 -0700

    Remove extraneous argLine variable
    
    (cherry picked from commit 18440df470ac61251aa9a2b6bc31fe6c96c50358)

diff --git a/pom.xml b/pom.xml
index c9fc6a0..1e82f99 100644
--- a/pom.xml
+++ b/pom.xml
@@ -570,8 +570,7 @@
             <reuseForks>false</reuseForks>
             <testFailureIgnore>${surefire.testFailureIgnore}</testFailureIgnore>
             <forkedProcessTimeoutInSeconds>${surefire.timeout}</forkedProcessTimeoutInSeconds>
-            <!--Allocate some direct memory for direct memory tests-->
-            <argLine>${hbase-surefire.argLine} ${argLine}</argLine>
+            <argLine>${hbase-surefire.argLine}</argLine>
             <redirectTestOutputToFile>${test.output.tofile}</redirectTestOutputToFile>
             <systemPropertyVariables>
               <jacoco-agent.destfile>target/jacoco.exec</jacoco-agent.destfile>
@@ -1901,7 +1900,7 @@
               <groupId>org.apache.maven.plugins</groupId>
               <artifactId>maven-surefire-plugin</artifactId>
               <configuration>
-                <argLine>${hbase-surefire.cygwin-argline} ${argLine}</argLine>
+                <argLine>${hbase-surefire.cygwin-argline}</argLine>
                 <systemProperties>
                   <property>
                     <name>java.net.preferIPv4Stack</name>
